pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -28,6 +28,8 @@ const (
 	// This should remain in the default role bindings for the NodeAdmin role
 	LegacyMasterKubeletAdminClientUsername = "system:master"
 	MasterKubeletAdminClientUsername       = "system:openshift-node-admin"
+
+	InfraNodeBootstrapServiceAccountName = "system:" + NodeBootstrapRoleName
```

Comment:
@liggitt thanks sorted out with @enj that in restroing the names I ended up reversing the role and the account name ... @tiran is going to fix it